### PR TITLE
Adding redis cloud pro to quick start

### PR DESCRIPTION
**Pull request by K-Jo**
Please hold back merging this PR.  Thanks

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -20,10 +20,17 @@ To see RedisGraph in action, visit [Demos](https://github.com/RedisGraph/RedisGr
...
 ## Quickstart
 
+1. [Redis Cloud](#rediscloud)
 1. [Docker](#docker)
-2. [Build](#building)
-3. [Start](#loading-redisgraph-into-redis)
-4. [Use from any client](#using-redisgraph)
+1. [Build](#building)
+1. [Start](#loading-redisgraph-into-redis)
+1. [Use from any client](#using-redisgraph)
```
Comment (swilly22) [inline docs/index.md:27]: How come each line is numbered as `1.` ?

Comment (K-Jo) [reply docs/index.md:27]: it's a more convenient way of creating ordered lists.

Comment (K-Jo) [reply docs/index.md:27]: It renders perfect. https://github.com/RedisGraph/RedisGraph/blob/45590ce5725ad7c5d731d6fdaadbf528994f503d/docs/index.md In the idea of enhancing things every time you touch them :-)

File: docs/index.md [hunk 2]
```diff
@@ -20,10 +20,17 @@ To see RedisGraph in action, visit [Demos](https://github.com/RedisGraph/RedisGr
...
 ## Quickstart
 
+1. [Redis Cloud](#redis-cloud)
 1. [Docker](#docker)
-2. [Build](#building)
-3. [Start](#loading-redisgraph-into-redis)
-4. [Use from any client](#using-redisgraph)
+1. [Build](#building)
+1. [Start](#loading-redisgraph-into-redis)
+1. [Use from any client](#using-redisgraph)
```
Comment (swilly22) [inline docs/index.md:27]: Why each line starts with `1.` ?

**Comment by K-Jo** [comment]
@gkorland can you cherry-pick to relevant branches?

**Pull request merged**